scripts/istio-images-mirror

```diff
@@ -25,8 +25,14 @@ library/nginx rancher/nginx 1.17.4-alpine
 jenkins/jnlp-slave rancher/jenkins-jnlp-slave 3.40-1
 thanosio/thanos rancher/thanos v0.10.1
 quay.io/jetstack/cert-manager-controller rancher/jetstack-cert-manager-controller v0.8.1
-quay.io/calico/node rancher/calico-node v3.10.2
-quay.io/calico/cni rancher/calico-cni v3.10.2
-quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.10.2
-quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.10.2
-us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.6.0
\ No newline at end of file
+quay.io/calico/node rancher/calico-node v3.12.0
+quay.io/calico/cni rancher/calico-cni v3.12.0
+quay.io/calico/ctl rancher/calico-ctl v3.12.0
+quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.12.0
+quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.12.0
+us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.6.0
+gcr.io/google_containers/k8s-dns-kube-dns rancher/k8s-dns-kube-dns 1.15.9
+gcr.io/google_containers/k8s-dns-dnsmasq-nanny rancher/k8s-dns-dnsmasq-nanny 1.15.9
+gcr.io/google_containers/k8s-dns-sidecar rancher/k8s-dns-sidecar 1.15.9
+weaveworks/weave-kube rancher/weaveworks-weave-kube 2.6.0
```

Comment:
Where are the weavework images used? It doesn't seem to be in Dockerhub?

Reply:
weaveworks is never used under rancher repo. Here is the issue: https://github.com/rancher/rancher/issues/22954